README.md

```diff
@@ -88,7 +88,7 @@ Feel free to [create an issue](https://github.com/EddieJaoudeCommunity/EddieBot/
 
 ### → Logging
 
-Logging will happen to the console as well as to the Discord `bot` channel.
+Logging will happen to the console as well as to the discord `bot` channel.
```

Comment:
`Discord` should capitals because it is a name

Reply:
Same with `Docker`, unless it is part of a command

Reply:
Okay